pkg/binlog/common/replication.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"time"
+
+	"github.com/siddontang/go-mysql/replication"
+)
+
+var (
+	// max reconnection times for binlog syncer in go-mysql
+	maxBinlogSyncerReconnect = 60
+
+	slaveReadTimeout      = 1 * time.Minute  // slave read binlog data timeout, ref: https://dev.mysql.com/doc/refman/8.0/en/replication-options-slave.html#sysvar_slave_net_timeout
```

Comment:
Maybe we can use `slaveReadTimeout` to replace `eventTimeout` in _syncer/streamer_controller.go_ and _relay/reader/reader.go_?

Reply:
replaced in 5f79104